modules/installation-configuration-parameters.adoc

```diff
@@ -608,9 +638,16 @@ accounts for the dramatically decreased machine performance.
 |Required if you use `controlPlane`. The name of the machine pool.
 |`master`
 
+ifndef::ibm-power-vs[]
```

Comment:
Remove lines 641, 645-650

Add` ,ibm-power-vs` to line 644

I wouldn't explictly add the comment to ensure that its set to ppc64le but stay in sync with the other arches.